src/Microsoft.DotNet.Interactive/Server/IPollingInputTextStream.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+namespace Microsoft.DotNet.Interactive.Server
+{
+    public interface IPollingInputTextStream : IInputTextStream
```

Comment:
An interface with a `void`  method on it is begging to be refactored out of existence.

Reply:
Done and deleted new interface